panther_lights/src/apa102.cpp

```diff
@@ -0,0 +1,102 @@
+#include <panther_lights/apa102.hpp>
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <linux/spi/spidev.h>
+#include <sys/ioctl.h>
+
+#include <cmath>
+#include <cstdint>
+#include <fstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace apa_102
+{
+APA102::APA102(const std::string device, const std::uint32_t speed, const bool cs_high)
+: device_(device), speed_(speed), fd_(open(device_.c_str(), O_RDWR))
+{
+  if (fd_ < 0) {
+    throw std::ios_base::failure(std::string("Failed to open ") + device_);
+  }
+
+  static std::uint8_t mode = SPI_MODE_3;
+  if (cs_high) {
+    mode |= SPI_CS_HIGH;
+  }
```

Comment:
```suggestion
  static std::uint8_t mode = cs_high ? SPI_MODE_3 : SPI_MODE_3 | SPI_CS_HIGH;
```
won't it work this way?